extra/test_release_tool.py

```diff
@@ -142,25 +142,67 @@ def test_version_of(capsys):
         capsys, ["--version-of", "gui", "--version-type", "git"], "master"
     )
 
-    # For an independent component, it should still accept docker/git type of the query
+    # Querying mender with version-type docker should error, since it maps to
+    # multiple containers and we don't know which one to choose.
     run_main_assert_result(capsys, ["--version-of", "mender"], "master")
+    try:
+        run_main_assert_result(
+            capsys, ["--version-of", "mender", "--version-type", "docker"], "master"
+        )
+    except:
+        pass
+    else:
+        pytest.fail("Expected exception!")
```

Comment:
An alternative using [pytest functionality](https://docs.pytest.org/en/6.2.x/reference.html#pytest.raises) would be:
```
with pytest.raises(RuntimeError):
    run_main_assert_result(
        capsys, ["--version-of", "mender", "--version-type", "docker"], "master"
    )
```

Reply:
Thanks, another nugget to remember! Changed.

Reply:
And by changed, I mean, I did it locally now, and will apply it together with the test re-enabling.